<ui version="4.0" >
 <class>unetbootin</class>
 <widget class="QWidget" name="unetbootin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>524</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>524</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>UNetbootin</string>
  </property>
  <widget class="QPushButton" name="FloppyFileSelector" >
   <property name="geometry" >
    <rect>
     <x>488</x>
     <y>100</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>false</bool>
   </property>
   <property name="toolTip" >
    <string>Select disk image file</string>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FloppyPath" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>100</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify a floppy or hard disk image file to load</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioFloppy" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify a floppy or hard disk image file to load</string>
   </property>
   <property name="text" >
    <string>Floppy Image:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioManual" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>150</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Manually specify a kernel and initrd to load</string>
   </property>
   <property name="text" >
    <string>Manual:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="KernelPath" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>150</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify a kernel file to load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="KernelFileSelector" >
   <property name="geometry" >
    <rect>
     <x>488</x>
     <y>150</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>false</bool>
   </property>
   <property name="toolTip" >
    <string>Select kernel file</string>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelkernel" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>150</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify a kernel file to load</string>
   </property>
   <property name="text" >
    <string>Kernel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelinitrd" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>180</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify an initrd file to load</string>
   </property>
   <property name="text" >
    <string>Initrd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="InitrdFileSelector" >
   <property name="geometry" >
    <rect>
     <x>488</x>
     <y>180</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="acceptDrops" >
    <bool>false</bool>
   </property>
   <property name="toolTip" >
    <string>Select initrd file</string>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InitrdPath" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>180</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify an initrd file to load</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioDistro" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select from a list of supported distributions</string>
   </property>
   <property name="text" >
    <string>Distribution:</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="distroselect" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>50</y>
     <width>182</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select from a list of supported distributions</string>
   </property>
   <property name="currentIndex" >
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="intromessage" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>10</y>
     <width>491</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Choose a mode of operation. If unsure, select an option under "Distribution"</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okbutton" >
   <property name="geometry" >
    <rect>
     <x>340</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelbutton" >
   <property name="geometry" >
    <rect>
     <x>430</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeldrive" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>41</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select the target drive to install to</string>
   </property>
   <property name="text" >
    <string>Drive:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="driveselect" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>260</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select the target drive to install to</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeldrive_2" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>260</y>
     <width>32</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select the installation target type</string>
   </property>
   <property name="text" >
    <string>Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeselect" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>260</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Select the installation target type</string>
   </property>
   <item>
    <property name="text" >
     <string>Hard Disk</string>
    </property>
   </item>
   <item>
    <property name="text" >
     <string>USB Drive</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="OptionEnter" >
   <property name="geometry" >
    <rect>
     <x>140</x>
     <y>210</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify parameters and options to pass to the kernel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeloption" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>210</y>
     <width>49</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>Specify parameters and options to pass to the kernel</string>
   </property>
   <property name="text" >
    <string>Option</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dverselect" >
   <property name="geometry" >
    <rect>
     <x>330</x>
     <y>50</y>
     <width>182</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select the distribution version&lt;/p>&lt;/body>&lt;/html></string>
   </property>
   <property name="currentIndex" >
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
